### PR TITLE
community rework

### PR DESCRIPTION
**Pull request by ChandlerDeKock**

### CODE REVIEW

File: docs/community/usingtokens.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+## Call Options Minting
+
+A call option gives the buyer the right - but not the obligation - to buy an asset at a specified price (strike) at a specific time period (expiry).  Call Options for UMA can be minted on the [MySynths platform](http://mysynths.finance). 
+
+Call Options may be an unfamiliar product to many DeFi users, so we would encourage people to read the articles explaining call options and how they have been implemented on UMA.  For a deeper dive, there is also an internal learning session on our YouTube which goes into more depth. 
+
+## uTVL Options
+
+If you had held over 10 UMA tokens, voted in our Voter Dapp or had minted an UMA product by 10th Feb 2021, you can claim the uTVL airdrop tokens from our [Claims Dapp](https://claim.umaproject.org/).  These tokens are redeemable for $UMA after the 30th June 2021 with their value depending on the Total Value Locked in the UMA protocol at that time. [uTVL updates](https://www.youtube.com/playlist?list=PLtbecZTA1xzyBxba9Zh8wmfqtA0YCwh6p) are available on our Youtube channel and also check out our FAQ (insert link to uTVL Options FAQ page)
```
Comment (mhairimca) [inline docs/community/usingtokens.md:26]: Insert link to uTVL faq

File: docs/community/usingtokens.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+title: Using your UMA tokens
+sidebar_label: How to use your UMA tokens?
+---
+
```
Comment (mhairimca) [inline docs/community/usingtokens.md:5]: Need a bit of an intro, how about

"So you've got some UMA....but what do you do with it?

Read on to find some suggestions for how you can use your UMA to participate in keeping our protocol secure, use it as collateral to mint dollar pegged expiring tokens or call options which entitle you to purchase more UMA at a fixed point int he future,   If you have held UMA for a while, its [worth checking](https://claim.umaproject.org/) if you received our uTVL airdrop which you can claim until the end of June.

File: docs/community/usingtokens.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+UMA tokens are accepted as collateral for minting Yield Dollars on the [UMAnji platform](https://umanji.xyz/) from Machina Labs.  UMA tokens are deposited as collateral to mint the yUMA21 token, which expires to exactly $1 on 31st December 2021. 
+
+Liquidity mining rewards are available for those who mint the token and use it to provide liquidity in the [yUMA21/USDC](https://info.uniswap.org/token/0x3108c33b6fb38efedaefd8b5f7ca01d5f5c7372d) liquidity pool on Uniswap.
+
+## Call Options Minting
+
+A call option gives the buyer the right - but not the obligation - to buy an asset at a specified price (strike) at a specific time period (expiry).  Call Options for UMA can be minted on the [MySynths platform](http://mysynths.finance). 
+
+Call Options may be an unfamiliar product to many DeFi users, so we would encourage people to read the articles explaining call options and how they have been implemented on UMA.  For a deeper dive, there is also an internal learning session on our YouTube which goes into more depth.
```
Comment (mhairimca) [inline docs/community/usingtokens.md:22]: insert links to Kevin's medium articles as well as the internal learning session on youtube.

**Comment by Gamaranto** [comment]
@mhairimca @ChandlerDeKock have the comments been resolved? What's needed to move this forward?

File: docs/community/community-overview.md [hunk 4]
```diff
@@ -1,13 +1,27 @@
...
+The UMA community consists of tokenholders, financial engineers, product designers developers and users. No matter which bucket you fall in, we're happy to have you join. We are a friendly bunch, and always willing to help people who have just discovered UMA make the most out of our innovations. 
 
-### Get involved
+Our community can usually be found hanging out on [Discord](https://discord.umaproject.org/),  where you can find details on upcoming proposals and votes, get support with using UMA tokens and products as well as exploring new ideas and how we can turn them into reality. 
+
+We also have a [Discourse](https://discourse.umaproject.org/), where improvements to our protocol are discussed as well as proposing ideas for new products. We also maintain a forum where UMA community members can showcase their talents, and product teams can offer bounties to our community for collaborative working. 
+
+You can also keep up to date with what is happening in UMA by following our Twitter; longer form explorations of UMA developments are available on our Medium, while our Youtube channel hosts presentations and tutorials in video format.
+
+## Featured initiatives
```
[the remark this answers is not in the document]
Comment (ChandlerDeKock) [reply docs/community/community-overview.md:14]: Not sure what you mean here. Move to the bottom of the current page or move to the page on next in sidebar list?

File: docs/community/community-overview.md [hunk 5]
```diff
@@ -1,13 +1,27 @@
...
+sidebar_label: Introduction to the UMA Community
 ---
 
-The UMA community consists of tokenholders, financial engineers, product designers and developers. No matter which bucket you fall in, we're happy to have you join. Please reference the resources below to get started.
+The UMA community consists of tokenholders, financial engineers, product designers developers and users. No matter which bucket you fall in, we're happy to have you join. We are a friendly bunch, and always willing to help people who have just discovered UMA make the most out of our innovations. 
 
-### Get involved
+Our community can usually be found hanging out on [Discord](https://discord.umaproject.org/),  where you can find details on upcoming proposals and votes, get support with using UMA tokens and products as well as exploring new ideas and how we can turn them into reality. 
+
+We also have a [Discourse](https://discourse.umaproject.org/), where improvements to our protocol are discussed as well as proposing ideas for new products. We also maintain a forum where UMA community members can showcase their talents, and product teams can offer bounties to our community for collaborative working.
```
[the remark this answers is not in the document]
Comment (ChandlerDeKock) [reply docs/community/community-overview.md:10]: Done

File: docs/community/community-overview.md [hunk 6]
```diff
@@ -1,13 +1,27 @@
...
+You can also keep up to date with what is happening in UMA by following our Twitter; longer form explorations of UMA developments are available on our Medium, while our Youtube channel hosts presentations and tutorials in video format.
+
+## Featured initiatives
+
+There is a lot happening in UMA and we would like to feature a few noteworthy programs we have ongoing at the moment. 
+There is an ongoing [Coinbase Earn](https://www.coinbase.com/earn/uma) campaign which gives a video overview of UMA as well the opportunity to collect a few tokens for watching explainer videos to allow you to get up to speed with one of the most innovative and fast moving protocols in the DeFi space.
+
+We encourage open, transparent and collaborative development within our community, which are a vital part of our success.  We also have a number of outstanding vacancies as well as a “talent referral options” program, which offers up to 1000 $UMA to anyone who refers suitably qualified candidates for any of our open positions. 
+
+### List of resources
```
[the remark this answers is not in the document]
Comment (ChandlerDeKock) [reply docs/community/community-overview.md:21]: Any reason why?

File: docs/community/communityguide.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
+---
```
[the remark this answers is not in the document]
Comment (ChandlerDeKock) [reply docs/community/communityguide.md:1]: Not entirely sure what you mean here either

File: docs/community/communityguide.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+title: Community Guides to Creating Synthetic Tokens on UMA
+sidebar_label: Community Guides to Creating Synthetic Tokens on UMA
+---
+
+We want to make the process of creating priceless contracts on UMA for synthetic tokens as easy as possible so that anyone is able to create one.  Community members have developed a wealth of resources and tools to make our ambition of Universal Market Access a reality.
+
+## Developing Your Idea
```
[the remark this answers is not in the document]
Comment (ChandlerDeKock) [reply docs/community/communityguide.md:8]: Done

File: docs/community/communityguide.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+## Using the Synths Builder
+
+Once your collateral type and price identifier have been approved for use by the UMA protocol, you now need to develop the contract for the synthetic asset. We have an interactive builder where you can spin up an UMA contract in minutes without any coding knowledge through a [point-and-click interface](http://syntheticbuilder.xyz/).
+
+A [video guide](https://www.youtube.com/watch?v=OwyGketRfhA&t=121s) to using the synths builder available, which can be used to create priceless financial contracts on either the Kovan testnet or on Mainnet. UMA runs a developer mining offering rewards to the developers of whitelisted contracts.  To be considered for this program, complete this [form](https://docs.google.com/forms/d/e/1FAIpQLSfhrCopRz7nUSbBQZB75j8yGqnXbOzRW68Oe6-uFuNVSvqrOQ/viewform), prior to mainnet deployment.
```
[the remark this answers is not in the document]
Comment (ChandlerDeKock) [reply docs/community/communityguide.md:24]: Done

File: docs/community/uTVLFAQ.md [hunk 10]
```diff
@@ -0,0 +1,42 @@
+---
+title: uTVL FAQ
+sidebar_label: uTVL FAQ
+---
+
+### What does KPI mean?
```
[the remark this answers is not in the document]
Comment (ChandlerDeKock) [reply docs/community/uTVLFAQ.md:6]: Done some changes, can you have a look

File: docs/community/uTVLFAQ.md [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+---
+
+### What does KPI mean? 
+KPI stands for Key Performance Indicator. 
+
+### What is a KPI option? 
+A KPI option is a financial product that becomes more valuable if a certain KPI the issuer cares about is met. These are intended to incentivize option holders to work towards meeting those KPIs.
+
+### What is uTVL-0621? 
+uTVL-0621 is the UMA's first KPI option product. Its value depends on the total value locked (TVL) in the UMA protocol by the end of June 2021.
```
[the remark this answers is not in the document]
Comment (ChandlerDeKock) [reply docs/community/uTVLFAQ.md:13]: Added

**Comment by ChandlerDeKock** [comment]
@mhairimca I added your suggested comments. Please have another look? 

There are 1 or 2 outstanding items I need a bit of clarity on

File: docs/community/community-overview.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (mhairimca) [reply docs/community/community-overview.md:14]: move to the next page in the sidebar list.

File: docs/community/community-overview.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (mhairimca) [reply docs/community/community-overview.md:21]: They are linked in above in narrative form

File: docs/community/communityguide.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (mhairimca) [reply docs/community/communityguide.md:1]: The content from this word doc
https://docs.google.com/document/d/1sR_7kl91xwS2OjxvLDHImKFs-XM9UTjZu_c9Xxys9aI/edit

File: docs/community/uTVLFAQ.md [hunk 12]
```diff
@@ -0,0 +1,42 @@
...
+2. Click Connect Wallet to Claim Options 
+3. Select a wallet provider and follow instructions 
+
+### I believe I am eligible but the web app says I'm not. What should I do? 
+1. First, make sure that you're using the right address 
+2. Then search for that address in this [list](https://github.com/UMAprotocol/token-distribution/blob/master/outputs/governance_recipients.json). 
+3. If your address isn't on the list but you still really, pinky promise think you're eligible, share it in a public channel of the UMA Discord server and we'll try to help you
+
+### I got my options. What do I do now? 
+You can now either hold your uTVL options and help the community drive up the uTVL of UMA to increase their value or sell them on SushiSwap.  You can also provide liquidity to this pool and receive liquidity provider fees.
```
[the remark this answers is not in the document]
Comment (mhairimca) [reply docs/community/uTVLFAQ.md:32]: (hyperlink to Sushiswap)``

File: docs/community/communityguide.md [hunk 13]
```diff
@@ -0,0 +1,28 @@
...
+
+### Developing Your Idea
+
+Everything begins with an idea.  A great place to look at whether someone is already developing in the area that you are interested in, or to flesh out your own idea is the UMA Discourse [Ideas Bank](https://github.com/Madtodist/UMIPs/blob/master/UMIPs/Big%20Mac%20UMIPv1.md). 
+
+There you can get feedback from other community members around both the concept and technicalities of the derivative to allow you to start the process of building your synthetic asset and making it available for minting. 
+
+### Writing An UMIP
+
+A synthetic asset contract needs to know two things - the collateral type that the asset will be backed by, and the reference price of the synthetic asset. Note that “Price” in this context does not necessarily mean a monetary value, as synthetic assets can track anything.
```
Comment (mhairimca) [inline docs/community/communityguide.md:16]: Add at end of sentance

"Templates for UMIPs are now available on the [Discourse](https://discourse.umaproject.org/).  Starting a new topic in one of the UMIP template categories, will bring up a preformatted template to use as a base"

**Pull request merged**